joomla/joomla-cms
# Add Count to Tags Overview

Pull Request to improve the information about the used Tags in the website by adding a new **Tagged Items column**. The Tagged Items column shows a counter of the amount of items that use that tag. It's sortable so you can easily get an overview about the most or less used tags.

### Summary of Changes
In the Back-end under Components > Tags you'll see an overview of all created Tags.

![tags-before](https://user-images.githubusercontent.com/1217850/51256213-20acbb00-19a5-11e9-893c-33d985beb359.png)

### Testing Instructions

- Add  some tags to some items (Articles or other items). 
- Check the Components > Tags overview
- Install the patch
- Check again the Components > Tags overview

### Expected result
A new column "**Tagged Items**" is available. You can click on the "**Tagged Items**" header to order the column. You can select the order also via the "Sort Table By:" dropdown on the top right.

![tags-after](https://user-images.githubusercontent.com/1217850/51259210-fd393e80-19ab-11e9-9ee4-27a6b6da8520.png)
